peregrine/resources/submission/__init__.py

```diff
@@ -133,12 +135,34 @@ def generate_schema_file(graphql_schema):
     with open(query_file, 'r') as f:
         query = f.read()
 
-    with open(schema_file, 'w') as f:
-        result = graphql_schema.execute(query)
-        data = {'data': result.data}
-        if result.errors:
-            data['errors'] = [err.message for err in result.errors]
-        json.dump(data, f)
+    try:
+        with open(schema_file, 'w') as f:
```

Comment:
It might be cool to add a check that if the `schema.json` already exists, and is not locked, then assume somebody else generated, and skip re-generating it ...